src/OnboardingSPA/pages/Steps/DesignThemeStyles/Menu/index.js

```diff
@@ -88,12 +92,6 @@ const StepDesignThemeStylesMenu = () => {
 		setPattern( patternsResponse?.body );
 		setGlobalStyles( globalStylesResponse?.body );
 		setSelectedStyle( currentData.data.theme.variation );
-		if ( '' === currentData.data.theme.variation ) {
```

Comment:
This is a gray area, I wanted to stay consistent with tracking actual user selections instead of the defaults. I'm pretty sure if we start adding default data as part of normal selections, we will skew a large amount of data to the default style. It's better to leave as is, until we either have a new action for default style or are fine to track defaults as well.